src/include/migraphx/program.hpp

```diff
@@ -126,6 +126,9 @@ struct program
     friend bool operator==(const program& x, const program& y);
     friend bool operator!=(const program& x, const program& y) { return !(x == y); }
 
+    std::shared_ptr<std::vector<std::pair<float, float>>> int8_quant_params =
```

Comment:
These should not be stored in the program. This should be stored in an auxiliary data structure.